src/core/config.py

```
@@ -14,12 +14,23 @@
 # limitations under the License.
 #
 
-"""Centralised logging configuration for the security tooling."""
+"""Runtime configuration – GitHub Actions environment detection and logging setup."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)